site/docs/v1/tech/apis/system.adoc

```diff
@@ -108,46 +108,35 @@ link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas
 [.endpoint]
 .URI
 --
-[method]#GET# [uri]#/api/system/log/export``?dateTimeSecondsFormat=\{dateTimeSecondsFormat\}&lastNBytes=\{lastNBytes\}&zoneId=\{zoneId\}``#
+[method]#GET# [uri]#/api/system/log/export``?dateTimeSecondsFormat=\{dateTimeSecondsFormat\}&includeArchived=true&lastNBytes=\{lastNBytes\}&zoneId=\{zoneId\}``#
 --
 
 When calling the API using a `GET` request you will send the export criteria on the URL using request parameters.
 
 ==== Request Parameters
 
-[.api]
-[field]#dateTimeSecondsFormat# [type]#[String]#  [optional]#Optional# [default]#defaults to **[see description]**#::
-The format string used to format the date and time columns in the export result.
-+
-When this parameter is omitted a default format of `M/d/yyyy hh:mm:ss a z` will be used. See the https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html[DateTimeFormatter patterns] for additional examples.
-
-[field]#lastNBytes# [type]#[Long]# [optional]#Optional# [default]#defaults to `65,536`#::
-The number of bytes to retrieve from the end of each of the system logs. When this value is `-1`, the entire log will be downloaded.
-
-[field]#zoneId# [type]#[String]# [optional]#Optional# [default]#defaults to **[see description]**#::
-The link:/docs/v1/tech/reference/data-types#time-zone[time zone] used to calculate the current time and build the filename.
-+
-For example:
-+
-[quote]
-`America/Denver` or `US/Mountain`
-+
-&nbsp;
-+
-When this parameter is omitted the configured default report time zone will be used. See [field]#reportTimezone# in the link:/docs/v1/tech/apis/system[System Configuration API].
+include::docs/v1/tech/apis/_system-logs-request-body.adoc[]
 
 [.api-authentication]
 link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Export the System Logs
 [.endpoint]
 .URI
 --
-[method]#POST# [uri]#/api/system/system-log/export#
+[method]#POST# [uri]#/api/system/log/export#
 --
 
 When calling the API using a `POST` request you will send the export criteria in a JSON request body.
 
+==== Request Body
+
 include::docs/v1/tech/apis/_system-logs-request-body.adoc[]
 
+[source,json]
```

Comment:
We usually include the same jsong in the include file. See https://github.com/FusionAuth/fusionauth-site/blob/master/site/docs/v1/tech/apis/_consent-request-body.adoc for example.

Any reason we can't do the same here?

Reply:
The include file is used for both the GET and POST, but only the POST has the request body so it's moved out into the including file, under the POST section.

Reply:
Ah, makes total sense. Thanks for explaining.